### DISCUSSION TITLE
Issue with Node 12.22.0

### DISCUSSION THREAD

**Discussion by SteeleCoale** [percy/cli#861] [Support] [answered]
We are using a docker Image that runs node 12.22.0 and noticed a problem today after we tried to use @percy/core@npm:1.0.1. 
We are downgrading our Percy package for now, but thought you might like to know in the meantime. Thanks!

"➤ YN0007: │ @percy/core@npm:1.0.1 must be built because it never has been before or the last one failed
➤ YN0009: │ @percy/core@npm:1.0.1 couldn't be built successfully (exit code 1, logs can be found here: /private/var/folders/45/fc2cmny174z7kx_z6blrffsm0000gq/T/xfs-58938596/build.log)
➤ YN0000: └ Completed in 1s 301ms
➤ YN0000: Failed with errors in 1s 685ms
jvila@Julio-V-MacBook-Pro liquid-smoke % cat /private/var/folders/45/fc2cmny174z7kx_z6blrffsm0000gq/T/xfs-58938596/build.log
# This file contains the result of Yarn building a package (@percy/core@npm:1.0.1)
# Script name: postinstall

file:///Users/jvila/projects/liquid-smoke/node_modules/@percy/core/post-install.js:6
await import('./dist/install.js').then(install => install.chromium());
^^^^^

SyntaxError: Unexpected reserved word
at Loader.moduleStrategy (internal/modules/esm/translators.js:140:18)
at async link [(internal/modules/esm/module_job.js:42:21)"

**Comment by Robdel12** [comment, marked as answer]
Hey @SteeleCoale! Percy CLI 1.x requires Node 14+ now. This is since Node 12 is EOL at the end of April of '22. This is highlighted at the top of the 1.0 discussion in the breaking changes section: https://github.com/percy/cli/discussions/854

**Comment by SteeleCoale** [comment]
Well that makes total sense. Thank you, and sorry for the extra noise.

**Reply by Robdel12** [reply]
No worries, it's unfortunate it fails in that way and [not the nice error we have built in. 😭](https://github.com/percy/cli/blob/master/packages/cli/bin/run.cjs#L4-L8) (We'll look into seeing how we can clean that up)
